packages/ingest/src/url-path-template.js

```diff
@@ -33,6 +33,9 @@ function evaluateOperation(name, args) {
   case 'extractPath': {
     return path.dirname(valueStr);
   }
+  case 'firstDefined': {
```

Comment:
lodash has a method defaultTo(value, defaultValue), and defaultTo is probably a better name.

Reply:
Need to add the function to https://github.com/nasa/cumulus/blob/master/docs/workflows/workflow-configuration-how-to.md?plain=1#L238

Reply:
Yup, makes sense. https://github.com/nasa/cumulus/pull/4021/commits/48fe4c4777eee5c438b3270a49de982b6d8f28dc